test/smoke-fort-dev/flang-568012/doit.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export OMP_TARGET_OFFLOAD=DISABLED
```

Comment:
This could also be done without a wrapper script by adding the following to the Makefile with:
RUNENV += OMP_TARGET_OFFLOAD=DISABLED